config/src/test/java/org/springframework/security/config/annotation/authentication/NamespaceJdbcUserServiceTests.java

```diff
@@ -114,6 +114,8 @@ void configure(AuthenticationManagerBuilder auth, DataSource dataSource) throws
 				.usersByUsernameQuery("select principal,credentials,true from users where principal = ?")
 				// jdbc-user-service@authorities-by-username-query
 				.authoritiesByUsernameQuery("select principal,role from roles where principal = ?")
+				// jdbc-user-service@change-password-query
```

Comment:
This test started failing when implementing the password update interface, which prompted me to tweak the test so the queries matched the schema.